src/Components/Blazor/Build/src/targets/StaticWebAssets.targets

```diff
@@ -0,0 +1,30 @@
+<Project>
+
+  <PropertyGroup>
+    <ResolveStaticWebAssetsInputsDependsOn>
+      $(ResolveStaticWebAssetsInputsDependsOn);
+      _RemoveBlazorWwwrootFromStaticWebAssets;
+    </ResolveStaticWebAssetsInputsDependsOn>
+  </PropertyGroup>
+
+
+  <Target Name="_RemoveBlazorWwwrootFromStaticWebAssets">
```

Comment:
The naming of this target seems odd. Why is the name talking about `Wwwroot` when the implementation doesn't say anything about that? The implementation appears to remove everything from `StaticWebAsset` if it doesn't have a `SourceType`. Could you add something (e.g., a comment, or change the name of the task) to clarify why this matches up with the task name?

Reply:
Updated the name of the target to better reflect what it does